rcos_io/views/meetings.py

```diff
@@ -99,6 +101,36 @@ def detail(meeting_id: str):
     )
 
 
+@bp.route("/<meeting_id>/host")
+@mentor_or_above_required
+@login_required
+def host(meeting_id: str):
```

Comment:
Does `open` make more sense here than `host`?

Reply:
I agree, I can change it based on which direction we take in the comment below.

Reply:
Fixed with latest changes.